accessibility/common-remediations.md

```diff
@@ -66,12 +70,24 @@ a:focus {
 }
 ```
 
+### Visual order doesn't match source order
+
+#### What's the problem?
+
+CSS positioning has been used so that the visual order of interactive elements doesn't match the HTML source order. 
+
+#### Who's affected by the problem?
 
+* Keyboard users
+* Screen reader users
 
-### Visual order is not followed with source code
-Just try to follow the visual order of elements while coding in HTML, don’t adjust elements to take positions with CSS, then this is confusing to keyboard users who expect to tab forwards and backwards in a linear sequence.
+#### Why's it a problem?
 
-As a rule, try tabbing through your pages every so often just to make sure you haven't accidentally messed up the tab order. 
+Keyboard users (including screen reader users) expect to tab forwards and backwards in a linear sequence. Switching the visual order of elements can confuse these users or cause them to activate the wrong control. 
```

Comment:
I understand what "expect to tab forwards and backwards in a linear sequence" means but I was wondering if other people will understand. Would it be clearer to say something like "(roughtly) from left to right and top to bottom for left-to-right languages" or something along those lines?

Reply:
any better now?

Reply:
🙆‍♂